src/main/java/org/jenkinsci/test/acceptance/plugins/ssh_slaves/SshSlaveLauncher.java

```diff
@@ -25,6 +27,19 @@
     public final Control retries = control("maxNumRetries");
     public final Control javaPath = control("javaPath");
     public final Control jvmOptions = control("jvmOptions");
+    public final Control hostKeyVerificationStrategy = control("/");
+
+
+    //<select class="setting-input dropdownList" path="/launcher/"><option value="0">Known hosts file Verification Strategy</option><option value="1">Manually provided key Verification Strategy</option><option value="2">Manually trusted key Verification Strategy</option><option value="3">Non verifying Verification Strategy</option></select>
+
+    public enum HostKeyVerificationStrategy {
+        Known_hosts_file(0), Manually_provided(1), Manually_trusted(2), Non_verifying(3);
```

Comment:
This will be a maintenance burden if we ever add or reorder those strategies. Consider using the ATH feature of associating selections with a `Descriptor.id` or `.displayName`.